example.config.yml

```diff
@@ -7,6 +7,10 @@ pihole_enable: true
 pihole_hostname: pihole
 pihole_timezone: America/Chicago
 pihole_password: "change-this-password"
+# Block Apple's iCloud Private Relay by default
+# - Prevent Apple devices from bypassing Pi-hole
+# - See also: Pi-hole configuration 'BLOCK_ICLOUD_PR'
+pihole_block_icloud_pr: true
```

Comment:
Just a question—is the current default in Pi-hole (if undefined) `true`? If not, I would be okay adding this feature in, but would not like to override Pi-hole's default _by default_.

Reply:
Thanks for looking into it! :)

Yes - the overridden value is equal to the default value (==`true`) -> Private Relay is blocked by default.

The motivation of this PR was that I (as a noob user in terms of DNS filters, etc. ... that configured the Pi-hole as DNS for its (WLAN-)router) instantly ran into the problem that my Apple devices showed a warning that I have to deactivate Private Relay in order to use the (WLAN-)connection. The way that I saw to fix that (being able to use Private Relay for Safari on my Apple devices and the features of pi-hole DNS for everything else) was to edit the template, which is not optimal as it is part of the source control (effectively forcing me to make a fork).

I thought that the configuration would be a good addition for what I suspected to be part of the main audience of this project (people who do not want to setup everything on their own because they do not need to configure every advanced setting available).

I understand if you have concerns about overriding the default value by default with the default value. :D

- Do you have an idea about an alternative way to control the value of that environment var without creating a fork of your project?
- Or is it possible expose the setting without setting a value?